ddtrace/context.py

```diff
@@ -40,6 +41,7 @@ def __init__(
         sampling_priority=None,  # type: Optional[float]
         meta=None,  # type: Optional[_MetaDictType]
         metrics=None,  # type: Optional[_MetricDictType]
+        lock=None,  # type: Optional[forksafe.ResetObject[threading.RLock]]
```

Comment:
Can't just do `Optional[forksafe.RLock]`?

Reply:
`forksafe.RLock` is a function not a type

Reply:
ah right, forgot about how the forksafe module worked